### PR TITLE
Added branding settings for hostListVC

### PR DESCRIPTION
**Pull request by harry528tt**
@bhariharan @trooper2013

### CODE REVIEW

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m [hunk 1]
```diff
@@ -167,6 +166,11 @@ - (void)viewWillAppear:(BOOL)animated {
     [self resizeContentForPopover];
     // style navigiation bar
     [super viewWillAppear:animated];
+    
+    SFSDKLoginViewControllerConfig *config = [SFUserAccountManager sharedInstance].loginViewControllerConfig;
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m]: @harry528tt Shouldn't this still be within the `if` block?

Comment (harry528tt) [reply libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m]: @bhariharan Actually I don't think it's necessary. The if condition at line 132 is for checking "rightBarButtonItem"; But the tintColor I set here is not only for that, but also for other navBarItems.

**Comment by harry528tt** [comment]
retest this please

**Comment by brandonpage** [comment]
@harry528tt Github pages take a while to index and actually display the results.  280 tests ran with 3 failures, but they look like flappers to me.

**Comment by harry528tt** [comment]
@brandonpage thanks for the confirmation. @trooper2013 @bhariharan I moved the setting branding part from viewWillAppear to viewDidLoad.

**Pull request merged**